src/Core/AdminFeatures/Providers/CreateProviderCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Bit.Core.AdminFeatures.Providers.Interfaces;
+using Bit.Core.Entities.Provider;
+using Bit.Core.Enums.Provider;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+
+namespace Bit.Core.AdminFeatures.Providers;
```

Comment:
As mentioned above, please move to `Bit.Commercial.Core.ProviderFeatures` namespace and folder.

Reply:
@eliykat should this be `Bit.Commercial.Core.Provider`? We removed the `Features` suffix in secrets manager since it's more a team and less a feature.

Reply:
I didn't realise we'd removed the `Features` suffix, so yes I agree. CC @r-tome

Reply:
I tried moving the command to the namespace `Bit.Commercial.Core.Provider` but then the `Provider` class always needed the full namespace to be strongly typed everywhere so I moved the command to `Bit.Commercial.Core.Providers` instead.